README.md

```diff
@@ -114,43 +163,276 @@ encoding as JSON. Since Python is a dynamic language, when decoding via
+    # Tell it to complete
+    await handle.signal(GreetingWorkflow.complete_with_greeting)
+    # Wait and return result
+    return await handle.result()
 ```
 
 Some things to note about the above code:
 
-* This creates/uses the same client that is used for starting workflows
+* This uses the `GreetingWorkflow` from the previous section
+* The result of calling this function is `"Aloha, my name!"`
+* `id` and `task_queue` are required for running a workflow
+* `client.start_workflow` is typed, so MyPy would fail if `"my name"` were something besides a string
+* `handle.signal` is typed, so MyPy would fail if `"Aloha"` were something besides a string or if we provided a
+  parameter to the parameterless `complete_with_greeting`
+* `handle.result` is typed to the workflow itself, so MyPy would fail if we said this `create_greeting` returned
+  something besides a string
+
+#### Invoking Activities
+
+* Activities are started with non-async `workflow.start_activity()` which accepts either an activity function reference
+  or a string name.
+* A single argument to the activity is positional. Multiple arguments are not supported in the type-safe form of
+  start/execute activity and must be supplied via the `args` keyword argument.
+* Activity options are set as keyword arguments after the activity arguments. At least one of `start_to_close_timeout`
+  or `schedule_to_close_timeout` must be provided.
+* The result is an activity handle which is an `asyncio.Task` and supports basic task features
+* An async `workflow.execute_activity()` helper is provided which takes the same arguments as
+  `workflow.start_activity()` and `await`s on the result. This should be used in most cases unless advanced task
+  capabilities are needed.
+* Local activities work very similarly except the functions are `workflow.start_local_activity()` and
+  `workflow.execute_local_activity()`
+
+#### Invoking Child Workflows
+
+* Child workflows are started with async `workflow.start_child_workflow()` which accepts either a workflow run method
+  reference or a string name. The arguments to the workflow are positional.
+* A single argument to the child workflow is positional. Multiple arguments are not supported in the type-safe form of
+  start/execute child workflow and must be supplied via the `args` keyword argument.
+* Child workflow options are set as keyword arguments after the arguments. At least `id` must be provided.
+* The `await` of the start does not complete until the workflow has confirmed to be started
+* The result is a child workflow handle which is an `asyncio.Task` and supports basic task features. The handle also has
+  some child info and supports signalling the child workflow
+* An async `workflow.execute_child_workflow()` helper is provided which takes the same arguments as
+  `workflow.start_child_workflow()` and `await`s on the result. This should be used in most cases unless advanced task
+  capabilities are needed.
```

Comment:
Advanced things are just signaling, or something else? (Wondering whether it's possible to be more specific)

Reply:
Yes, I can be more specific. This README is intentionally sparse because I have a hard time justifying writing all of this that I am going to turn around and delete. That's why it's just bullets, limited examples, etc. All of this you're reviewing and I'm writing will be deleted.

Depending on time, it's only value may be to help @rachfop get off the ground w/ the basics.